### ISSUE TITLE
initial research

### ISSUE THREAD

**Issue by yamatokataoka** [yamatokataoka/learning-from-human-preferences#7]
- collect information for Replication of Deep Reinforcement Learning from Human Preferences (Christiano et al, 2017).

**Comment by yamatokataoka** [comment]
- [Learning from Human Preferences](https://openai.com/blog/deep-reinforcement-learning-from-human-preferences/)
- [Deep reinforcement learning from human preferences](https://arxiv.org/abs/1706.03741)

**Comment by yamatokataoka** [comment]
implementations
- https://github.com/mrahtz/learning-from-human-preferences
- https://github.com/HumanCompatibleAI/learning-from-human-preferences
- https://github.com/nottombrown/rl-teacher

**Comment by yamatokataoka** [comment]
final training code would look like this using [rl-replicas](https://github.com/yamatokataoka/reinforcement-learning-replications)

```
class Sampler():
```

**Comment by yamatokataoka** [comment]
When do we collect preferences?
- At the beginning of training the reward predictor is pre-trained
- labels are fed in at a rate decaying inversely with the number of timesteps
